docs/topics/migrations.txt

```diff
@@ -455,7 +455,7 @@ the file in the right place, suggest a name, and add dependencies for you)::
 Then, open up the file; it should look something like this::
 
     # Generated by Django A.B on YYYY-MM-DD HH:MM
-    from django.db import migrations, models
+    from django.db import migrations
```

Comment:
Thanks, I forgot the behavior changed in a7bc00e17bebe34cdf23f744937b3c386ee73d81.